notebooks/metrics_demo.ipynb

```diff
@@ -113,7 +123,10 @@
    "source": [
     "## Adjusted random index\n",
     "This is a metrics which is used to measure the similarity between two data clusters, and it is adjusted to take into consideration the chance grouping of elements.\n",
-    "For more information on Adjusted random index please refer to: https://en.wikipedia.org/wiki/Rand_index"
+    "For more information on Adjusted random index please refer to: https://en.wikipedia.org/wiki/Rand_index\n",
```

Comment:
I really like the extended descriptions of the metrics. I think we should also extend these descriptions to include the datasets, extended evaluation criteria, and models as well.